RELEASE_PROCESS.md

```diff
@@ -0,0 +1,268 @@
+# Podman Releases
+
+## Overview
+
+Podman (and podman-remote) versioning is mostly based on semantic-versioning
+standards.
+Significant versions
+are tagged, including *release candidates* (`rc`).
+All relevant **major** releases have their own branches.  The **latest**
```

Comment:
Technically it's relevant minor releases - 2.0 and 2.1 and 2.2 were all on separate upstream branches.